crates/datasources/src/native/access.rs

```diff
@@ -125,7 +130,7 @@ impl NativeTableStorage {
     }
 
     pub async fn delete_table(&self, table: &TableEntry) -> Result<()> {
-        let prefix = format!("databases/{}/tables/{}", self.db_id, table.meta.id);
+        let prefix = format!("databases/{}/tables/{}", self.db_id, table.meta.name);
```

Comment:
```suggestion
        let prefix = format!("databases/{}/tables/{}", self.db_id, table.meta.id);
```

To avoid breakage on table rename.

Reply:
oh i was just using this for testing, forgot to revert.